File: lat3_PHP/Processor.php
<?php

class Processor{
    private $Name = "";
    private $Price = 0;
    
    public function __construct($Name = "", $Price = 0){
        $this->Name = $Name;
        $this->Price = $Price;
    }

    public function setName($Name){
        $this->Name = $Name;
    }

    public function getName(){
        return $this->Name;
    }

    public function setPrice($Price){
        $this->Price = $Price;
    }

    public function getPrice(){
        return $this->Price;
    }

    public function printProcessor(){
        echo "Processor Name    : ". $this->getName() ."<br>";
        echo "Processor Price   : ". $this->getPrice() ."<br>";
    }

}  
?>

File: lat3_java/processor.java
class processor{
    private String name;
    private int price;
    
    public processor() {
    }
    
    public processor(String name, int price) {
        this.name = name;
        this.price = price;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public void printProcessor(){
        System.out.print("Processor Name  : ");
        System.out.println(getName());
        System.out.print("Processor price : ");
        System.out.println(getPrice());
    }
    
}